lib/approval_request_listener.rb

```diff
@@ -0,0 +1,64 @@
+require "manageiq-messaging"
+
+class ApprovalRequestListener
+  SERVICE_NAME = "platform.approval".freeze
+  CLIENT_AND_GROUP_REF = "catalog-api-worker".freeze
```

Comment:
@syncrou Should this be an approval-catalog-api-worker.

Reply:
I don't think the name really matters (correct me if I'm wrong?), so I think the change is a fine idea 👍

Reply:
@eclarizio Yes, I believe that value is internal only.